src/basilisp/lang/runtime.py

```diff
@@ -2638,3 +2647,8 @@ def get_compiler_opts() -> CompilerOpts:
     v = Var.find_in_ns(CORE_NS_SYM, sym.symbol(COMPILER_OPTIONS_VAR_NAME))
     assert v is not None, "*compiler-options* Var not defined"
     return cast(CompilerOpts, v.value)
+
+
+def is_reiterable_iterable(x: Any) -> bool:
+    """Return ``true`` if x is a re-iterable Iterable object."""
+    return isinstance(x, Iterable) and iter(x) is not x
```

Comment:
Could you move this function with the vaguely related group of functions with under the comment section "Runtime Support"?

Probably could just put it right under where you included `to_iterator_seq`.

Reply:
relocated, thanks